substrate/frame/revive/src/vm/evm/instructions/contract.rs

```diff
@@ -50,16 +56,27 @@ pub fn create<'ext, const IS_CREATE2: bool, E: Ext>(context: Context<'_, 'ext, E
 	popn!([value, code_offset, len], context.interpreter);
 	let len = as_usize_or_fail!(context.interpreter, len);
 
+	// TODO: We do not charge for the new code in storage. When implementing the new gas:
```

Comment:
can we link to the issue if we have it already

Reply:
done